src/core/adapters/youtrack.test.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @jest-environment node
+ */
+
+import { JSDOM } from "jsdom";
+
+import scan from "./youtrack";
+
+const html = {
+  issuepage: (type?: string) => `
```

Comment:
Is this structure supposed to leave room for quickly adding support for other pages in the future? Sweet 🍬 

From the product page it looks like there's a Jira-like backlog view where one quick-views issues. That could could probably be a candidate 👀

Reply:
Yep, exactly. I took inspiration from the existing Jira adapter when I whipped up this PR. It seemed like there might be other pages to extract tickets from and/or subtle differences between the internal JetBrains YouTrack instance and cloud-hosted instances. I can imagine that there's on-premise instances out there too… To get started I decided to just go for this one case.